src/test/java/scm/address/logic/commands/ImportCommandTest.java

```diff
@@ -48,6 +53,72 @@ public void execute_existingFile_success() {
         assertCommandSuccess(importCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_importingFromCsv_success() throws CommandException {
+        HashSet<File> curHashSet = new HashSet<>();
+        curHashSet.add(new File(ADDRESS_BOOK_CSV_PATH));
+
+        Model testModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        ImportCommand importCommand = new ImportCommand(curHashSet);
+        importCommand.execute(testModel);
+
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        expectedModel.addPerson(JAMES);
+
+        assertEquals(testModel, expectedModel);
+    }
+
+
+    @Test
+    public void retrievePersonsFromFile_validJsonFile_success() throws CommandException {
+        HashSet<File> curHashSet = new HashSet<>();
+        curHashSet.add(new File(ADDRESS_BOOK_PATH));
+
+        Model testModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        ImportCommand importCommand = new ImportCommand(curHashSet);
+        importCommand.execute(testModel);
+
+        assert true;
```

Comment:
Not sure whether `assert true` is the right way to do this or `assertTrue(true)` is right, since the former is a Java assertion and not a JUnit assertion, I think?

Reply:
Yep! I fixed it! Thanks for the bug detection!